examples/website/scenegraph-layer/app.js

```diff
@@ -64,43 +97,94 @@ export class App extends Component {
           id: 'scenegraph-layer',
           data,
           pickable: true,
-          sizeScale: 10,
-          scenegraph:
-            'https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Models/master/2.0/Duck/glTF-Binary/Duck.glb',
+          sizeScale: 20,
+          scenegraph: MODEL_URL,
           getPosition: d => [
             d[DATA_INDEX.LONGITUDE] || 0,
             d[DATA_INDEX.LATITUDE] || 0,
             d[DATA_INDEX.GEO_ALTITUDE] || 0
           ],
-          getOrientation: d => [0, 90 + (d[DATA_INDEX.TRUE_TRACK] || 0), 90],
+          getOrientation: d => [
+            this._verticalRateToAngle(d),
+            // TODO: Fix this direction
+            (d[DATA_INDEX.TRUE_TRACK] || 0) - 180,
+            90
+          ],
           getTranslation: [0, 0, 0],
-          getScale: [1, 1, 1]
+          getScale: [1, 1, 1],
+          transitions: {
+            getPosition: REFRESH_TIME * 0.9
+          },
+          onHover: ({object}) => this.setState({hoverObject: object})
         })
       ];
     }
 
     return [];
   }
 
+  _renderHoverObject() {
+    const [icao24 = '', callsign = '', originCountry = ''] = this.state.hoverObject;
+    const verticalRate = this.state.hoverObject[DATA_INDEX.VERTICAL_RATE] || 0;
+    const velocity = this.state.hoverObject[DATA_INDEX.VELOCITY] || 0;
+    const track = this.state.hoverObject[DATA_INDEX.TRUE_TRACK] || 0;
+    return (
+      <Fragment>
+        <div>Unique ID: {icao24}</div>
+        <div>Call Sign: {callsign}</div>
+        <div>Country: {originCountry}</div>
+        <div>Vertical Rate: {verticalRate} m/s</div>
+        <div>Velocity: {velocity} m/s</div>
+        <div>Direction: {track}</div>
+      </Fragment>
+    );
+  }
+
   render() {
     const {viewState, controller = true, baseMap = true} = this.props;
 
     return (
-      <DeckGL
-        layers={this._renderLayers()}
-        initialViewState={INITIAL_VIEW_STATE}
-        viewState={viewState}
-        controller={controller}
-      >
-        {baseMap && (
-          <StaticMap
-            reuseMaps
-            mapStyle={MAPBOX_STYLE}
-            preventStyleDiffing={true}
-            mapboxApiAccessToken={MAPBOX_TOKEN}
-          />
-        )}
-      </DeckGL>
+      <Fragment>
+        <DeckGL
+          layers={this._renderLayers()}
+          initialViewState={INITIAL_VIEW_STATE}
+          viewState={viewState}
+          controller={controller}
+        >
+          {baseMap && (
+            <StaticMap
+              reuseMaps
+              mapStyle={MAPBOX_STYLE}
+              preventStyleDiffing={true}
+              mapboxApiAccessToken={MAPBOX_TOKEN}
+            />
+          )}
+        </DeckGL>
+        <div
```

Comment:
The information panel is rendered and styled by the website, see https://github.com/uber/deck.gl/blob/master/website/src/components/demos/geojson-demo.js#L27